app/services/subcase-is-approved.js

```diff
@@ -28,4 +28,40 @@ export default class SubcaseIsApprovedService extends Service {
     }
     return false;
   }
+
+  async isRetracted(subcase) {
+    const meeting = await subcase?.requestedForMeeting;
+
+    if (meeting?.isFinal) {
+      const retractedDecisionResultCode = await this.store.findRecordByUri(
+        'decision-result-code',
+        CONSTANTS.DECISION_RESULT_CODE_URIS.INGETROKKEN
+      );
+
+      const nrDecisionActivities = await this.store.count('decision-activity', {
+        'filter[subcase][:id:]': subcase.id,
+        'filter[decision-result-code][:id:]': retractedDecisionResultCode.id,
+      });
```

Comment:
Do we really need this extra call to get the `retractedDecisionResultCode` concept?

Wouldn't `'filter[decision-result-code][:uri:]': CONSTANTS.DECISION_RESULT_CODE_URIS.INGETROKKEN,` also work?
(see https://github.com/mu-semtech/mu-cl-resources#special-filters )

Reply:
Ah yes, we can.
I copied this from a similar one that had 2 id's (since 2 URI's dont work)
But with only 1, uri is fine.